### ISSUE TITLE
Newline braces have redundant ident

### ISSUE THREAD

**Issue by CodeJjang** [ionutvmi/sublime-jsfmt#45]
I get the following no matter what, tried importing all esformatter confs and zeroing them but nothing help:

```
return (
    <div>
    <Comment author='Aviad'/>
    <Comment author='Bar Inline'/>
    </div>
    );
```

Instead:

```
return (
    <div>
    <Comment author='Aviad'/>
    <Comment author='Bar Inline'/>
    </div>
);
```

Seems like a bug?

**Comment by ionutvmi** [comment]
Look like it was reported here https://github.com/millermedeiros/esformatter/issues/465

**Comment by wildeyes** [comment]
Can this be closed?

**Comment by ionutvmi** [comment]
Hi @wildeyes , @CodeJjang ,

esformatter has not released a version with that fix included.
However I was able to manually apply the patch to the esformatter version bundled with sublime-jsfmt.

The fix should be applied in the 1.0.11 release. 
Please test it and let me know if it fixes the issue.
